client/package-lock.json

```diff
@@ -9,7 +9,7 @@
       "resolved": "https://registry.npmjs.org/@ant-design/colors/-/colors-4.0.5.tgz",
       "integrity": "sha512-3mnuX2prnWOWvpFTS2WH2LoouWlOgtnIpc6IarWN6GOzzLF8dW/U8UctuvIPhoboETehZfJ61XP+CGakBEPJ3Q==",
       "requires": {
-        "tinycolor2": "1.4.1"
```

Comment:
Please, uncommit this file. We are not making updates to this file unless you are adding a new library. Let me know if you have any quetions or if you need any help. 

Otherwise, looks good! :D

Reply:
@tr1503 Any chance you can uncommit the changes to this file? - I could do it from my end if it is better.

Reply:
Yeah, can you do it for me? Thanks!

Reply:
@idjevm Hi :) Can you help us please?

Reply:
Done @doheekim93  :)